# DP: diff between the standalone branch and the oldgit branch
# DP: (note that master has no delta against standalone)

diff --git a/post-receive-email b/post-receive-email
index abde17e..335ab59 100755
--- a/post-receive-email
+++ b/post-receive-email
@@ -405,7 +405,7 @@ generate_create_branch_email()
 	    git rev-list --parents --topo-order --reverse --stdin $newrev | \
 	    head -n 1 |&
 	read -p commit parent otherparents
-	git -c core.quotePath=false diff-tree $diffopts \
+	git diff-tree $diffopts \
 	    "${parent:-$(:|git hash-object -t tree --stdin)}" "$newrev"
 }
 
@@ -584,7 +584,7 @@ generate_update_branch_email()
 	# non-fast-forward updates.
 	echo ""
 	echo "Summary of changes:"
-	git -c core.quotePath=false diff-tree $diffopts $oldrev..$newrev
+	git diff-tree $diffopts $oldrev..$newrev
 }
 
 #
